tests/output-eve-anomaly-01/test.yaml

```diff
@@ -1,9 +1,7 @@
 requires:
   features:
     - HAVE_LIBJANSSON
-
-  files:
-    - src/output-json-anomaly.c
+  min-version: 5.0
```

Comment:
Should `min-version` be `major.minor` or `major.minor.patch` with `5.0` (`5.0.0`)? Both are used; are `5.0` and `5.0.0` equivalent at this stage?

Reply:
Seems like it won't matter after https://github.com/OISF/suricata-verify/pull/158 is merged. We can be flexible about minor and patch if only we are OK with them being set to 0.